### PR TITLE
Database positions

### PR DESCRIPTION
**Pull request by AndyTWF**
- Move controller positions to the database from the flat files.
- Move airfield ownership to the database from the flat files.
- Delete some SIDs that aren't in the sector file anymore.
- Add handoff orders for SIDs
- Supersedes #55

### CODE REVIEW

File: app/Console/Commands/GetDeletedSidsFromSectorFile.php [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+        foreach ($removedSids as $sid) {
+            $this->info($sid);
+        }
+    }
+
+    private function processPlugin() : array
+    {
+        $sidArray = [];
+        $sids = Sid::all();
+        $sids->each(function (Sid $sid) use (&$sidArray) {
```
Comment (AxonC) [inline app/Console/Commands/GetDeletedSidsFromSectorFile.php]: Could this not be a `->map` function?

File: app/Console/Commands/GetDeletedSidsFromSectorFile.php [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+     *
+     * @param string $line
+     * @return bool
+     */
+    private function exitingSidSection(string $line) : bool
+    {
+        return substr($line, 0, 4) === 'STAR';
+    }
+
+    private function parseSidData(string $line) : string
```
Comment (AxonC) [inline app/Console/Commands/GetDeletedSidsFromSectorFile.php]: Could you document the array indexes here?

File: app/Http/Controllers/AirfieldController.php [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        $this->airfieldService = $airfieldService;
+    }
+
+    /**
+     * @param Request $request
+     * @return JsonResponse
+     */
+    public function getAllAirfields(Request $request) : JsonResponse
+    {
+        $airfields = $request->get('controllers') === '1'
```
Comment (AxonC) [inline app/Http/Controllers/AirfieldController.php]: Is this a boolean or a string? Could it be worth casting this to boolean?

File: app/Services/HandoffService.php [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    {
+        $handoffs = Handoff::all();
+        $handoffArray = [];
+
+        $handoffs->each(function (Handoff $handoff) use (&$handoffArray) {
+            $handoffArray[] = array_merge(
+                $handoff->toArray(),
+                [
+                    'controllers' =>
+                        $handoff->controllers()->orderBy('order')->pluck('controller_position_id')->toArray()
```
Comment (AxonC) [inline app/Services/HandoffService.php]: Is it worth eager-loading the controllers relationship here?

File: database/migrations/2019_09_14_094017_update_gatwick_departures.php [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+class UpdateGatwickDepartures extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        $gatwick = Airfield::where('code', 'EGKK')->firstOrFail()->id;
```
Comment (AxonC) [inline database/migrations/2019_09_14_094017_update_gatwick_departures.php]: Minor issue - what happens if `ModelNotFoundException` is thrown (I know it is a major edge-case and will only likely get run once, but I'd hate the deployment to fail)?

File: app/Console/Commands/GetDeletedSidsFromSectorFile.php (hunk 1 above)
Comment (AndyTWF) [reply app/Console/Commands/GetDeletedSidsFromSectorFile.php]: Done!

File: app/Console/Commands/GetDeletedSidsFromSectorFile.php (hunk 2 above)
Comment (AndyTWF) [reply app/Console/Commands/GetDeletedSidsFromSectorFile.php]: Done!

File: app/Http/Controllers/AirfieldController.php (hunk 3 above)
Comment (AndyTWF) [reply app/Http/Controllers/AirfieldController.php]: Originally intended to come in as a string in the URL like `https://ukcp.vatsim.uk/airfield?controllers=1`

File: database/migrations/2019_09_14_094017_update_gatwick_departures.php (hunk 5 above)
Comment (AndyTWF) [reply database/migrations/2019_09_14_094017_update_gatwick_departures.php]: I've sorta gone with the view that if I haven't accounted for all the required data, that we'll have bigger problems! That said, I've gone for the fail fast approach so we can quickly find what went wrong. Is there another, better way we could do this?

File: app/Services/HandoffService.php (hunk 4 above)
Comment (AndyTWF) [reply app/Services/HandoffService.php]: Very eager!

File: database/migrations/2019_09_14_094017_update_gatwick_departures.php (hunk 5 above)
Comment (AxonC) [reply database/migrations/2019_09_14_094017_update_gatwick_departures.php]: I think this works actually.

**Pull request merged**